src/dotty/tools/dotc/transform/LazyVals.scala

```diff
@@ -308,9 +312,9 @@ class LazyVals extends MiniPhaseTransform with SymTransformer {
             } else { // need to create a new flag
               offsetSymbol = ctx.newSymbol(companion.moduleClass, (StdNames.nme.LAZY_FIELD_OFFSET + id.toString).toTermName, Flags.Synthetic, defn.LongType).enteredAfter(this)
               val flagName = (StdNames.nme.BITMAP_PREFIX + id.toString).toTermName
-              val flagSymbol = ctx.newSymbol(claz, flagName, containerFlags, defn.LongType).enteredAfter(this)
+              val flagSymbol = ctx.newSymbol(claz, flagName, containerFlags, defn.LongType).entered
```

Comment:
Why `entered` and not `enteredAfter`. Are we sure we enter this in a fresh scope. If not this would lead
to symbols mysteriously popping up in scopes from later phases. Think "back to the future" or "terminator".